sympy/utilities/lambdify.py

```diff
@@ -528,6 +528,12 @@ def sub_args(args, dummies_dict):
                 dummies = Dummy()
                 dummies_dict.update({args : dummies})
                 return str(dummies)
+            elif isinstance(args ,(Function, Derivative)):
```

Comment:
The `Function` here does nothing, as it's caught by the `if` clause above.

Reply:
What do I need to do? Can you please help?
@asmeurer , @moorepants , please review.